sdk/identity/identity/review/identity-api-review.md

```diff
@@ -0,0 +1,137 @@
+## API Report File for "@azure/identity"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { AccessToken } from '@azure/core-http';
+import { GetTokenOptions } from '@azure/core-http';
+import { ServiceClientOptions } from '@azure/core-http';
+import { TokenCredential } from '@azure/core-http';
+
+export { AccessToken }
+
+// @public
+export class AggregateAuthenticationError extends Error {
+    constructor(errors: any[]);
+    errors: any[];
+}
+
+// @public
+export const AggregateAuthenticationErrorName = "AggregateAuthenticationError";
+
+// @public
+export class AuthenticationError extends Error {
+    constructor(statusCode: number, errorBody: object | string | undefined | null);
+    readonly errorResponse: ErrorResponse;
+    readonly statusCode: number;
+}
+
+// @public
+export const AuthenticationErrorName = "AuthenticationError";
+
+// @public
+export class AuthorizationCodeCredential implements TokenCredential {
+    constructor(tenantId: string, clientId: string, clientSecret: string | undefined, authorizationCode: string, redirectUri: string, options?: IdentityClientOptions);
+    getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken | null>;
+    }
+
+// @public
+export type BrowserLoginStyle = "redirect" | "popup";
+
+// @public
+export class ChainedTokenCredential implements TokenCredential {
+    constructor(...sources: TokenCredential[]);
+    getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken | null>;
+    }
+
+// @public
+export class ClientCertificateCredential implements TokenCredential {
+    constructor(tenantId: string, clientId: string, certificatePath: string, options?: IdentityClientOptions);
+    getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken | null>;
```

Comment:
GetToken in TokenCredential needs to be updated to take TokenRequestContext, a structure with the scopes on it, as well as the parent request id.